tools/fetch_elfutils.sh

```diff
@@ -68,6 +68,18 @@ patch -p1 < "${SCRIPT_DIR}/elfutils.patch"
 if [[ "$(basename "${C_COMPILER}")" == clang* ]]; then
   export CFLAGS="-Wno-xor-used-as-pow -Wno-gnu-variable-sized-type-not-at-end -Wno-unused-but-set-parameter"
 fi
+
+# Detect musl
+# If so, then set the DFNM_EXTMATCH macro to a harmless value.  We don't use the affected
+# codepaths, so it's OK (TODO: can we just disable compilation of those units?)
+MUSL_LIBC=$(ldd /bin/ls | grep 'musl' | head -1 | cut -d ' ' -f1 || true)
```

Comment:
You could have taken build recipe / patches from: 
https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/main/elfutils/APKBUILD

Reply:
Fixed, thanks